packages/odyssey-react-mui/src/ui-shell/SideNav/SideNavLogo.tsx

```diff
@@ -51,11 +58,20 @@ const SideNavLogo = ({
       aria-label={optionalProps.ariaLabel}
       data-se="sidenav-header-logo"
       href={optionalProps.href}
+      onClick={optionalProps.onClick}
     >
       {logo}
     </StyledLogoLink>
   ) : (
-    <StyledLogoContainer role="presentation">{logo}</StyledLogoContainer>
+    <StyledLogoContainer
+      role="presentation"
+      aria-label={optionalProps.ariaLabel}
+      data-se="sidenav-header-logo"
+      onClick={optionalProps.onClick}
+      isLogoClickable={!!optionalProps.onClick || optionalProps.href}
```

Comment:
I like stating intent with code. `Boolean()` says get the boolean value of this.

Using `!!` says "I don't want the opposite of that thing when cast as a boolean". It's not clear when coming across the code because it's a hack that just happens to work.

```suggestion
      isLogoClickable={Boolean(optionalProps.onClick) || optionalProps.href}
```